### DISCUSSION TITLE
tine podcast: 03 – Willkommen Pino

### DISCUSSION THREAD

**Discussion by pschuele** [tine20/tine20#7415] [General]
Es gibt einen Neuzugang bei tine: Pino. Pino ist der Name der tine Business Edition 2022.11, die gerade frisch erschienen ist.

Entsprechend dreht sich sich in dieser Episode alles um das Major Release. Philipp Schüle, Senior Core Entwickler, erklärt die Neuerungen. Kleine Erleichterungen für Admins, die neue Integration des Broadcasthubs und größere Umwälzungen in einigen Anwendungen jenseits des klassischen Groupware-Portfolios. Ein Zombie hat einen Auftritt und Phil spoilert schon mal eine große Neuentwicklung für das kommende Jahr! In Kurzform nachlesen könnt Ihr die Neuerungen auch in den [Release Notes](https://www.tine-groupware.de/wp-content/uploads/2022/11/tine_ReleaseNotesFlyer_2022_DE_final.pdf).

Gebt uns gerne Feedback zu dieser Episode. Habt Ihr vielleicht Themen, über die wir unbedingt mal sprechen sollten oder andere Anregungen? Wenn es Euch gefallen hat, hinterlasst uns doch eine positive Bewertung im Podcast-Portal Eurer Wahl.

Neue Episoden erscheinen immer am vierten Mittwoch des Monats – am besten richtet Ihr Euch schon mal einen Serientermin in tine ein.

https://www.tine-groupware.de/podcast/03-willkommen-pino/

**Comment by lab-at-nohl** [comment]
Thank you for the insights. And greetings to you in regard to the episode before...

I waited for a year to not miss the next steps in daylight (not in nightlies). If anybody is looking for the **community release** see [pschuele's links...](https://github.com/tine20/tine20/discussions/7357#discussioncomment-4253986).

**Comment by lab-at-nohl** [comment]
Is the BroadcastHub open source? 

It is not _PHP_ but a combination of _npm_ and _redis_, I guess. Can you, maybe, add a small wiki page? Including a configuration sample to use the docker image (firewall: udp?) but the local redis server would be nice.

**Reply by pschuele** [reply]
the broadcasthub is not yet on github - maybe we'll open source it soon.
you can get the docker image here -> https://hub.docker.com/r/tinegroupware/broadcasthub
I'll ask in our team about a short howto in the wiki.

**Reply by pschuele** [reply]
hey,
here is the broadcasthub-readme file:
[broadcasthub_README.md](https://github.com/tine20/tine20/files/10319985/broadcasthub_README.md)

**Reply by pschuele** [reply]
We now have a github repo for the broadcasthub: https://github.com/tine-groupware/broadcasthub/

See this readme for more information: https://github.com/tine-groupware/broadcasthub/blob/master/TINEDOCS.md

**Reply by lab-at-nohl** [reply]
Thanks. I have a `location` in nginx to reverse proxy websockets, configured **dotenv** and `config.inc.php` like this:

```
REDIS_URL=/run/redis/localhost.sock
REDIS_CHANNEL=broadcasthub
AUTH_TIMEOUT=5000
WS_PORT=8080
#WS_PORT=/run/broadcasthub.socket
ENABLE_MULTITENANCY_MODE=false
TINE20_JSON_API_URL=http://leaptest/
TEST_WS_URL=http://leaptest/broadcasthub/
DEBUG_DEFAULT_LOGGING=on
```

and

```php
  'broadcasthub' =>
  array(
    'active' => true,
    'url' => '//leaptest/broadcasthub/',
    'redis' =>
    array(
      'host' => '/run/redis/localhost.sock',
      'port' => -1,
      'pubsubname' => 'broadcasthub',
    ),
  ),
```

But I do not see any difference. Even in browser's console I cannot see a sign of Broadcasthub. What am I doing wrong?
